pkgs/by-name/cr/crowdsec-firewall-bouncer/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

see https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes

---

Not OP, but awesome! I missed that #390220 had been merged :D